tests/add.test.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import Task from '../src/task.js';
+import ToDoList from '../src/list.js';
+
+test('add one task to the list', () => {
+  const section = document.createElement('div');
+  const toDoList = new ToDoList(section, []);
+
+  const task = new Task('write a book', false, toDoList);
+
+  toDoList.add(task);
+
+  expect(toDoList.array.length).toBe(1);
+});
```

Comment:
- [ ] As stated in the project requirement Kindly Make sure you group your tests using the `describe()` method. because For better organizing your test cases, make sure you define your test blocks inside a `describe`.

![Capture](https://user-images.githubusercontent.com/93598189/207848782-a9364d00-7629-452c-94de-c5da194ebcb2.PNG)